SU-SWS/stanford_basic
# MRCD8-251 404 errors

# READY FOR REVIEW

# Summary
- Address 404 errors on MRC by adding missing assets to the /css folder in stanford_basic

# Needed By (Date)
- End of sprint

# Urgency
- Low

# Steps to Test
1. Checkout this branch
2. Verify paths match those configured in the css files

# Affected Projects or Products
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-251

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
